lib/mintel/dashboards/components/panels/cost-analysis.libsonnet

```diff
@@ -26,7 +26,7 @@ local seriesOverrides = import 'components/series_overrides.libsonnet';
       ],
       format='percent',
       gaugeMaxValue=100,
-      interval='',
+      interval='2m',
```

Comment:
do we set a `minInterval` in the commonPanel ? 

* If yes then don't need this one here
* If not let's set it there ?

Reply:
yes but we set a common interval of 1m

Reply:
Can't we use that rather than set 2m on each widget here ?

Reply:
I've removed this so that they use the common default of 1m.